peer_mentoring/tests/test_groups.py

```diff
@@ -25,23 +30,28 @@ def avatar(db):
     return avatar
 
 
-# Fixture creates a category instance
+# Fixture creates a category instance for group
 @pytest.fixture
 def category(db):
     return Category.objects.create(name="Test Category")
 
 
-# Fixture creates a user instance
+# Creates user instance
 @pytest.fixture
 def user(db):
-    return User.objects.create_user(username="testuser", password="12345")  # noqa: S106
+    return UserFactory()
+
+
+@pytest.fixture
+def user2(db):
```

Comment:
seems duplicated code, you already have it in fixtures.py

Reply:
Yes, it wasn't finding it otherwise, but corrected with the conftest.py